appveyor.yml

```diff
@@ -22,34 +25,61 @@
   environment:
     SKIP_SASS_BINARY_DOWNLOAD_FOR_CI: true
     matrix:
+      - nodejs_version: 6
+        GYP_MSVS_VERSION: 2015
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      - nodejs_version: 7
+        GYP_MSVS_VERSION: 2015
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      - nodejs_version: 8
+        GYP_MSVS_VERSION: 2015
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      - nodejs_version: 9
+        GYP_MSVS_VERSION: 2015
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      - nodejs_version: 10
+        GYP_MSVS_VERSION: 2015
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
       - nodejs_version: 11
+        GYP_MSVS_VERSION: 2015
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
       - nodejs_version: 10
-      - nodejs_version: 9
-      - nodejs_version: 8
-      - nodejs_version: 7
-      - nodejs_version: 6
+        GYP_MSVS_VERSION: 2017
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      - nodejs_version: 11
+        GYP_MSVS_VERSION: 2017
+        APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
 
   install:
-    - ps: Install-Product node $env:nodejs_version $env:platform
+    # https://www.appveyor.com/docs/lang/nodejs-iojs/#installing-any-version-of-nodejs-or-iojs
+    - ps: Update-NodeJsInstallation (Get-NodeJsLatestBuild $env:nodejs_version) $env:platform
     - node --version
     - npm --version
     - git submodule update --init --recursive
-    - npm install --msvs_version=2015
+    - npm install
 
   test_script: npm test
 
   before_deploy:
     # Save artifact with full qualified names of binding.node
     # (which we use in node-sass-binaries repo)
-    - ps: Get-ChildItem .\vendor\**\*.node | % { Push-AppveyorArtifact $_.FullName -FileName (($_.FullName.Split('\\') | Select-Object -Last 2) -join '_') }
-
+    - ps: >-
+        Get-ChildItem .\vendor\**\*.node | % {
+            ( $BindingName = $_.FullName ).Split('\\') |
+              Select-Object -Last 2 | Select-Object -First 1 } |
+                .{ process { (
+                  @( $BindingName,
+                        ( ( $_, "binding.node" ) -join '_' ) ),
```

Comment:
What does .pdb file represent ? I am not sure if they are even generated ?

Reply:
This was pulled from the upstream.  I was going to look into it as I am not sure, it only seems to be generated for windows.

Reply:
Are we ok with leaving this as is or should I revert? @junedkazi